pkg/common/resources/resources_test.go

```diff
@@ -1066,9 +1066,9 @@ func TestSubOnlyExistingNil(t *testing.T) {
 		}
 	}()
 	var r *Resource
-	r.SubOnlyExisting(nil)
-	if r != nil {
-		t.Errorf("sub nil resources did not return nil resource: %v", r)
+	result := SubOnlyExisting(r, nil)
+	if result != nil {
+		t.Errorf("SubOnlyExisting with nil resources did not return nil resource: %v", result)
```

Comment:
I think TestSubOnlyExistingNil() has been included in TestSubOnlyExisting, could we remove it?

Reply:
Sure, `TestSubOnlyExistingNil()` has been removed.